Makefile

```diff
@@ -27,19 +27,18 @@ MLRUN_DOCKER_IMAGE_PREFIX := $(if $(MLRUN_DOCKER_REGISTRY),$(strip $(MLRUN_DOCKE
 MLRUN_LEGACY_DOCKER_TAG_SUFFIX := -py$(subst .,,$(MLRUN_LEGACY_ML_PYTHON_VERSION))
 MLRUN_LEGACY_DOCKERFILE_DIR_NAME := py$(subst .,,$(MLRUN_LEGACY_ML_PYTHON_VERSION))
 
-
+.PHONY: help
 help: ## Display available commands
```

Comment:
we don't need to have the PHONY twice.
the decision is between: 
1. having them all in one place at the bottom of the file - the advantage is that this way the file has less "dirt", the disadvantage is that just like already happened, people add new targets, and forget to add them in the PHONY
2. having them next to each target - easier to maintain, but doesn't look good.

I tend to option 1 since we don't really add new targets that often

Reply:
Ah I forgot about that, apoligies.

2 >> 1 so removing from bottom of the file

Maintaining that list is really a b^%$. Already did some changes there, and adding/changing this is really annoying as there's nothing to help you correlated that magic list in the bottom with the actual blocks.
I don't think the "dirt" factor has any meaning in this context personally

Reply:
Welp, moving back to the list at the bottom of the file as per @Hedingber's request